en/docs/authentication-and-authorization/secure-api-access-with-azure-ad.md

```diff
@@ -17,16 +17,16 @@ This guide walks you through the following steps:
 Before you try out this guide, complete the following:
 
 -  [Configured Azure AD as an external IdP](../administer/configure-an-external-idp/configure-azure-ad-as-an-external-idp.md). 
```

Comment:
Should be "Configure Azure AD as an external IdP" right?

Reply:
done